cla_backend/apps/reports/urls.py

```diff
@@ -7,6 +7,7 @@
 urlpatterns = patterns(
     "",
     url(r"^api/exports/$", api.ExportListView.as_view(), name="exports"),
+    url(r"^api/exports/scheduled/$", api.ExportListView.as_view(scheduled=True), name="exports"),
```

Comment:
So, the `name` here can be used by Django for reverse url lookups ie getting a url from the url pattern name.
As you can see, we already have three url patterns that use the same `name` ("exports").
However, they all have different signatures in terms of url parameters, that is:

- reverse with name = exports -> api/exports/
- reverse with name = exports and a pk kwarg -> api/exports/<pk>/
- reverse with name = exports and a filename -> exports/download/<file_name>/

so depending what `reverse()` is called with, it's unambiguous which url pattern should be used.

However, if we add the scheduled list pattern with the same name, the reverse lookup would now be ambiguous. I think it would default to the first one, and it might not strictly matter for us in this case, but I'd prefer to avoid it with a unique (or, strictly, unique given its signature) `name`